modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -270,7 +323,7 @@ public void handle(VersionedUpdate update) {
 
             registerCatalog(catalog);
 
-            versionTracker.update(catalog.version(), null);
+            CompletableFuture.allOf(eventFutures.toArray(CompletableFuture[]::new)).thenRun(() -> versionTracker.update(version, null));
```

Comment:
`thenRun` is invoked only in case of successful completion, but what if any listener will throw an exception? We will get a frozen tests without notion about what went wrong. Let's at least log the exception